### PR TITLE
Add --prerelease

### LINKED ISSUE #33048: Missing --prerelease option
--prerelease option is not described in this page while you can find it with `dotnet tool update -h`
_--> determining whether to include pre-release packages._

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6a8171f4-8466-98ed-acb8-49563b864d16
* Version Independent ID: a1b7af2d-e2ba-bc4b-4d35-a8edbaa1249f
* Content: [dotnet tool update command - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-update)
* Content Source: [docs/core/tools/dotnet-tool-update.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-tool-update.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

---
[Associated WorkItem - 55447](https://dev.azure.com/msft-skilling/Content/_workitems/edit/55447)

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #33048

### CODE REVIEW

File: docs/core/tools/dotnet-tool-update.md [hunk 1]
```diff
@@ -84,6 +87,10 @@ The `dotnet tool update` command provides a way for you to update .NET tools on
 
   Do not cache packages and HTTP requests.
 
+- **`--prerelease`**
+
+  Include pre-release packages.
```
Comment (gewarren) [inline docs/core/tools/dotnet-tool-update.md:92]:
```suggestion
  Include pre-release versions.
```

**Review by gewarren — APPROVED** [review]
Left a suggestion for consideration. Also wondering if pre-release needs to be hyphenated on line 92?

File: docs/core/tools/dotnet-tool-update.md [hunk 2]
```diff
@@ -84,6 +87,10 @@ The `dotnet tool update` command provides a way for you to update .NET tools on
 
   Do not cache packages and HTTP requests.
 
+- **`--prerelease`**
+
+  Include pre-release versions.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-tool-update.md:92]:
```suggestion
  Include prerelease versions.
```

**Comment by tdykstra** [comment]
@gewarren Thanks for the review!

**Pull request merged**